### DISCUSSION TITLE
Microsoft Azure does not pass verification using My Student GitHub Account

### DISCUSSION THREAD

**Discussion by batuhanysv** [community/community#128329] [GitHub Education]
### Hi everyone,

Hi there,

A few months ago, I received a link to activate my Github Student Developer Pack from my educational institution. I activated it and it works with almost all services except Microsoft Azure.

The main problem is that I'm unable to complete the verification with Microsoft Azure to get student benefits.
When I try to go through the Azure for Students verification by opening this link:
https://signup.azure.com/studentverification?offerType=1
and using GitHub as a verification method I see this:
"
The account you are signed is not yet verified to access offer benefits.
...
We could not confirm your status as a student.
"
I also contacted Microsoft and a Microsoft employee asked me for information to verify my status as a student:

The purple box from (https://education.github.com/discount_requests/student_application) confirming my student status as approved.
Can you tell me what the issue is?

**Comment by yusufekorman** [comment]
same thing
